src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1349,6 +1349,10 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
             DISPTREERANGE(BlockRange(), condSelNode);
             JITDUMP("\n");
         }
+        else
+        {
+            assert(!"Embedded mask operation is not used anywhere.");
```

Comment:
I imagine this will trigger for minors

I would expect what we want to do is always do the transform to introduce CndSel, for correctness and to ensure we have a mask register to pass to emit

We then replace the use if one exists and otherwise mark the cndsel as unused, which is the more standard pattern if you look across TryGetUse calls